content/en/docs/advanced-tasks/unsafe-fail-forward-upgrades.md

```diff
@@ -0,0 +1,123 @@
+---
+title: "Opting into UnsafeFailForward upgrades"
+linkTitle: "Opting into UnsafeFailForward upgrades"
+weight: 4
```

Comment:
nit: It looks like there are more than 3 docs in this folder. But since we don't really care about the order of the documents in this folder it's not a blocker. 

Fyi the `weight` is mainly used to determine the order of appearance of the document